build.gradle

```diff
@@ -193,6 +194,15 @@ configure(projectsWithFlags('java')) {
     }
 }
 
+// Configure all Kotlin projects
+configure(projectsWithFlags('kotlin')) {
+    // Suppress the 'Class X is compiled by a pre-release version of Kotlin and
+    // cannot be loaded by this version of the compiler' error.
+    tasks.withType(KotlinCompile) { task ->
+        task.kotlinOptions.freeCompilerArgs += '-Xskip-prerelease-check'
```

Comment:
Should we move the option to? https://github.com/line/armeria/blob/11c6233d556f44a1aaf2a309e15ec399d53d2144/gradle/scripts/lib/kotlin.gradle#L6

Reply:
We can. Let me send out a PR there and merge it.

Reply:
https://github.com/line/gradle-scripts/pull/116